CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+cmake_minimum_required(VERSION 3.11)
+
+project(correctionlib VERSION ${CORRECTIONLIB_VERSION} LANGUAGES CXX)
+
+set(PKG_INSTALL "correctionlib") ## python package name
+
+include(FetchContent)
+FetchContent_Declare(pybind11
```

Comment:
Does this interfere at all with the pybind11 submodule brought in by `git clone --recursive`? Should we remove the submodule in favor of this method?

Reply:
No, this will actually use the submodule, but since that contains the pybind11 source (not an install directory) find_package doesn't work; FetchContent will run just enough of the cmake build of the submodule to use it as a dependency.